File: Language specifics/Language-specifics-Java/src/SumReversedNumbers.java
import java.util.Arrays;
import java.util.Scanner;

public class SumReversedNumbers {
    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);

        int[] numbers = Arrays.stream(scan.nextLine()
                .split("\\s+"))
                .mapToInt(Integer::parseInt)
                .toArray();
        long result = 0L;
        for (int i = 0; i < numbers.length; i++){
            int reversedNum = NumberReverse(numbers[i]);

            result += reversedNum;
        }

        System.out.println(result);
    }

    private static int NumberReverse(int num) {
        int reversed = 0;
        while(num != 0){
            reversed *= 10;
            reversed += num%10;
            num /= 10;
        }
        return reversed;
    }
}

File: Language specifics/Language-specifics-Java Script/SumReversedNumbers.js
function sumReversedNumbers(input){
    let numbers = input[0].split(' ');

    let sum = 0;

    for(let i=0; i<numbers.length; i++){
        let num =(numbers[i]);
        let reversed = parseInt(reverseNum(num));
        sum+=reversed;
    }

    console.log(sum);

    function reverseNum(num) {
        let reversed = "";

        for(let i=num.length-1;i>=0; i--) {
            reversed += num[i];
        }

        return reversed;
    }
}

//sumReversedNumbers(["123 234 12"]);
//sumReversedNumbers(["12 12 34 84 66 12"]);
//sumReversedNumbers(["120 1200 12000"]);